nav2_waypoint_follower/src/waypoint_follower.cpp

```diff
@@ -175,6 +175,9 @@ WaypointFollower::followWaypoints()
 
   rclcpp::WallRate r(loop_rate_);
 
+  auto start_time = this->now();
```

Comment:
start_time and last_wp_time isn't reset when the action is preempted with a new task

Reply:
Same with `result = std::make_shared<ActionT::Result>();` to reset the wp states if we have a new task. This isn't a bug you introduced, but just finding it now

Reply:
Do you want me to fix it in this PR? Or can I split that into its own?

Reply:
It should be fixed here